app/controllers/api/v1/accounts_controller.rb

```diff
@@ -13,9 +13,11 @@ def show
   end
 
   def follow
-    FollowService.new.call(current_user.account, @account.acct)
+    reblogs_arg = { reblogs: params[:reblogs] }
```

Comment:
This could be a private method

Reply:
In theory, yes? I already think it's not great because we're using it as an options argument as well as a return value (mostly for convenience and line size, because it's the same in both cases for the moment). I don't think it really has any purpose being a method, but I can make it one if you want.